main.js

```diff
@@ -1,12 +1,13 @@
 /* -----------------NAVBAR TOGGLE --------------*/
 // DOM  items
+const viewportWidth = window.innerWidth;
 const navbar = document.querySelector('#navbar');
 const burger = document.querySelector('#burger');
 
 let showMenu = false;
 
```

Comment:
![Screenshot from 2023-05-25 03-19-47](https://github.com/sergio-usma/First_capstone_project/assets/83042742/f20dbe49-3ee4-40c1-99ec-885dc526db6b)
- [ ] I think it would be better if your `Speakers` section sticks as much as possible to the one of the Bihance template on Mobile version, as it should only show two speakers by `Default` and show others when you click on the `See More` button. You can add that feature with JavaScript to match the requirements.
### Template Design
![Screenshot from 2023-05-25 03-19-04](https://github.com/sergio-usma/First_capstone_project/assets/83042742/e6491a3a-c3d0-4a3c-bffd-d2e2cccd8778)

### Your design
![Screenshot from 2023-05-25 03-25-35](https://github.com/sergio-usma/First_capstone_project/assets/83042742/2fd4cef6-eccc-4847-945e-81460f5370a5)

Reply:
Ok! I added the show more button.
![image](https://github.com/sergio-usma/First_capstone_project/assets/126805818/bdbdb708-e47c-4c14-b589-951f49fc49bd)